contrib/scalapblib/test/src/TutorialTests.scala

```diff
@@ -33,21 +34,19 @@ object TutorialTests extends TestSuite {
 
   object TutorialWithAdditionalArgs extends TutorialBase {
     object core extends TutorialModule {
-      override def scalaPBAdditionalArgs = Seq(
-        s"--zio_out=..."
-      )
+      override def scalaPBAdditionalArgs = T{Seq(
+        "--additional-test=..."
+      )}
```

Comment:
What's the point of changing this?

Reply:
I was written from the version before last scalapb pr and thinking "additional-test" might easier for other people to reasoning and not specific for zio, not something matters in term of code execution.